andela/ah-shakas-frontend
# #160577447  liking and disliking articles

## What does this PR do?
- Have the implementation for liking and disliking articles

## Description of Task to be completed?
 - implemented liking of articles
- implemented disliking articles

## How should this be manually tested?
- clone this repo `git clone https://github.com/andela/ah-shakas-frontend.git`
- cd into ah-shakas-frontend `cd ah-shakas-frontend` 
- checkout to this branch `git checkout ft-liking-articles-160577447 `
- run `yarn install`
- run `yarn start`
- login and visit any of the published articles
- you should be able to see a like and dislike icons that are clickable

## Any background context you want to provide?
- You should have the following in your .env
```
REACT_APP_BASE_URL=https://ah-shakas-staging.herokuapp.com/api
REACT_APP_UPLOAD_BASE_URL='https://api.cloudinary.com/v1_1/'
REACT_APP_CLOUD_NAME='dbk8ky24f'
REACT_APP_UPLOAD_PRESET='default'
```
## What are the relevant pivotal tracker stories?
[#160577447 ](https://www.pivotaltracker.com/n/projects/2198533/stories/160577447)
## Packages installed
None

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------


## Todos

## Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30701246/49013023-71195080-f18c-11e8-853c-804ba1ad330d.png)

## Questions:
